.github/workflows/mdspan-version-check.yml

```diff
@@ -1,7 +1,14 @@
 name: mdspan version check
 
 on:
-  workflow_call:
+  push:
```

Comment:
My 2 cents on this. Because of the limitations of these filters, we've sometimes used https://github.com/marketplace/actions/changed-files :wink:

Reply:
Do you typically have a more complex workflow?
Are you just mentioning that action for the record or are you saying that using it would improve the current PR?